monai/bundle/workflows.py

```diff
@@ -207,10 +208,20 @@ def __init__(
         **override: Any,
     ) -> None:
         super().__init__(workflow=workflow)
+        if config_file is not None:
+            _config_path = Path(ensure_tuple(config_file)[0])
+            if _config_path.is_file():
+                config_root_path = _config_path.parent
+            else:
+                raise FileNotFoundError(f"Cannot find the config file: {config_file}.")
```

Comment:
```suggestion
                raise FileNotFoundError(f"Cannot find the config file: {_config_path}.")
```

I see that this section only checks the first file so that it can get `config_root_path` but I think it should check that all config files exist or the class relies on the parser to do these checks which will make for less understandable errors. This way of getting `config_root_path` also assumes that all the config files are in the `configs` directory which isn't enforced elsewhere. They *should* be so I'd just note that we'd have to say any command line that refers to files outside this directory is technically wrong.

Reply:
Hi @ericspod, thanks for the suggestion, I just added some warnings and also checked whether all config files exist in this [commit](https://github.com/Project-MONAI/MONAI/pull/6835/commits/edb8768b9fb1ad8f0d39e4419f99373d3eaaf4b1). Could you please help review it, thanks!

Reply:
I think that looks good thanks!